e2e/test/scenarios/filters/filter-bulk.cy.spec.js

```diff
@@ -156,7 +156,7 @@ describe("scenarios > filters > bulk filtering", () => {
     filter();
 
     modal().within(() => {
-      cy.findByText("Product").click();
+      cy.findByText("Product").click({ force: true });
```

Comment:
There's a regression:

1. Create a question Sample DB -> Orders and visualize it
2. Click "Summarize" in chill mode
3. Click "Subtotal"
4. Move your mouse away

The binning strategy is not shown: [this PR](https://github.com/metabase/metabase/assets/6830683/106dcafa-9b6b-4d02-b15d-27780734b75f) vs [`master`](https://github.com/metabase/metabase/assets/6830683/a727a07f-6c47-453e-b1f4-ec60ad7f8b28)
It's only shown on hover.

Reply:
That's a regression in the base branch that. I've set up a [separate PR](https://github.com/metabase/metabase/pull/39244) for that since it has nothing to do with this PR.